src/components/RegistrationForm/Form/RegisterTerms/RegisterTerms.styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const StyledText = styled.p`
+  margin-bottom: 18px;
+  font-family: Gilroy-Medium, sans-serif;
+  font-size: 10px;
+  font-weight: 500;
+  line-height: 15px;
+  letter-spacing: 0;
+
```

Comment:
There is no reason to make an empty line

Reply:
This is a draft version of the code.